qemu/tests/stop_cont_vm_during_virtiofs.py

```diff
@@ -0,0 +1,123 @@
+import time
+import os
+import threading
+
+from virttest import utils_test, utils_misc, utils_disk
+from virttest import error_context
+from virttest import data_dir
+from provider import win_driver_utils
+from provider import virtio_fs_utils
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Stop/continue the vm during the virtiofs test.
+
+    1) Boot guest w/ virtiofs device.
+    2) For windows guest, enable driver verifier first.
+    3) Run virtiofs function test.
+    4) During steps 3, stop vm and then resume vm.
+    5) Memory leak check.
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    @error_context.context_aware
+    def stop_resume_vm_when_finding_file():
+        start_time = time.time()  # record the start time.
+        # run time expected, the unit is second.
+        max_run_time = params.get("max_run_time", 20)
```

Comment:
params.get_numeric, once you define max_run_time in cfg, you'll get an error below because you get a string instead of a number